test/performance/odp_packet_gen.c

```diff
@@ -145,6 +154,14 @@ static void print_usage(void)
 	       "  -t, --num_tx              Number of transmit threads. Default: 1\n"
 	       "  -n, --num_pkt             Number of packets in the pool. Default: 1000\n"
 	       "  -l, --len                 Packet length. Default: 512\n"
+	       "  -L, --len_range <min,max,bins>\n"
+	       "                            Random packet length. Specify the minimum and maximum\n"
+	       "                            packet lengths and the number of bins. To reduce pool size\n"
+	       "                            requirement the length range can be divined into even sized\n"
+	       "                            bins. Min and max size packets are always used and one size\n"
+	       "                            is selected from each other bin. Bin value of 0 means that\n"
+	       "                            each packet length is used. Comma-separated (no spaces).\n"
+	       "                            Overrides standard packet length option.\n"
```

Comment:
Number of bins documentation could be more clear e.g.:
"... Min and max size packets are always used and included into number of bins (bins >= 2) ..."